conda_libmamba_solver/mamba_utils.py

```diff
@@ -88,25 +88,20 @@ def init_api_context() -> api.Context:
     api_ctx = api.Context()
 
     # Output params
-    # We use this getattr() trick to guarantee backwards compatibility
-    # with libmambapy <=1.4.2; 1.4.3+ grouped some settings together
-    output_params = getattr(api_ctx, "output_params", api_ctx)
-    output_params.json = context.json
-    output_params.quiet = context.quiet
-    output_params.verbosity = context.verbosity
+    api_ctx.output_params.json = context.json
+    api_ctx.output_params.quiet = context.quiet
+    api_ctx.output_params.verbosity = context.verbosity
     api_ctx.set_verbosity(context.verbosity)
-    if output_params.json:
+    if api_ctx.output_params.json:
```

Comment:
We are cancelling some other json if json is True?

Reply:
We don't want libmamba to print JSON on its own. It's something only conda should do.